_editorial_policies/01_bylaws.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: single
+sidebar:
+  nav: policies_bylaws.md
+title: Bylaws of the Living Journal of Computational Molecular Science
+excerpt: Bylaws of LiveCoMS.
+permalink: /policies/livecoms_bylaws/
+---
+
+1. Organization Title and Publication
+
+   The Journal shall be called the Living Journal of Computational
+   Molecular Science, hereafter referred to as LiveCoMS.
+
+   LiveCoMS shall produce one publication, the Living Journal of
+   Computational Molecular Science, with articles released on a rolling
+   basis online. LiveCoMS provides a peer-reviewed home for manuscripts
+   which share best practices in molecular modeling and simulation. These
+   works are living documents, regularly updated with community input,
+   and can include living or perpetual reviews, tutorials, comparisons
+   between software packages, and other documents which aim to improve
+   the studies in the field and require ongoing updates.
+
+1. Editorial Board
+
+   The editorial board shall consist of three Managing Editors, any
+   number of Associate Editors, and a Section Lead Editor for each type
+   of document published in LiveCoMS, and any number of Associate
+   Editors. Section Lead Editors may be either Managing Editors or
+   Associate Editors.
+
+   1. Responsibilities
+       1. The Managing Editors coordinate the operation of the Journal and
+          the finances of the journal, maintain the online presence.  Tasks
+          may be divided up among managing editors as they see fit, and specific
+          individual tasks may be assigned to individual associate editors.
+       1. Section Lead Editors handles submitted manuscripts and assigns
+          all manuscripts to associate lead editors. They make decisions on
+          presubmission letters on about proposed manuscripts, in which they
+          may be advised by the associate editors. They make final decisions
+          based on recommendations from the associate editors, and communicate
+          them to the authors.
+       1. Associate Editors assign manuscripts to reviewers, collect
+          these reviews, and make recommendations to the Section Lead
+          Editors about acceptance, rejection, or requests for revision.
+
+   1. Selection process
+       1. Members of the editorial board will be elected as needed by a two-thirds vote of the editorial board.
+         1. Managing editors and section lead editors will be selected by a two-thirds vote of the
+            entire editorial board from by the other current members of the editorial board when there is a
+            vacancy in either capacity.
+         1. Vacancies in the Section Lead editors can be filled on a temporary basis by an associate editor selected unanimously by the managing editors until the position can be filled, for no more than three months.
```

Comment:
We need section leads always, as papers come in.  We can't do without.  I would imagine that either the board elects someone else, or the position gets filled temporarily again.   So, yes, pretty much what you say, except we expect that it will be filled.